features/support/env.rb

```diff
@@ -25,10 +25,12 @@ def setup_environment
 end
 
 
+# rubocop:disable MethodLength
 def initialize_environment
   # Create origin repo and set "main" as default branch
   create_repository :origin do
     run 'git symbolic-ref HEAD refs/heads/main'
+    configure_git 'user'
```

Comment:
origin shouldn't have a user. If you need a tag on the origin and not locally, have the coworker create it and push it up.